File: app-android/app/src/main/java/de/tutao/tutanota/alarms/EndType.java
package de.tutao.tutanota.alarms;

public enum EndType {
	NEVER, COUNT, UNTIL;


	public static EndType get(long value) {
		if (value > EndType.values().length - 1) {
			throw new IllegalArgumentException("Unknown EndType: " + value);
		}
		return EndType.values()[(int) value];
	}
}

File: app-android/tutashared/src/main/java/de/tutao/tutashared/alarms/EndType.kt
package de.tutao.tutashared.alarms

enum class EndType {
	NEVER,
	COUNT,
	UNTIL;

	companion object {
		operator fun get(value: Long): EndType {
			require(value <= values().size - 1) { "Unknown EndType: $value" }
			return values()[value.toInt()]
		}
	}
}